packages/yoastseo/src/parse/build/private/filterHelpers.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Filter helpers fort the html treefilter.
+ * Are all functions that return a callback with the element as only argument.
+ */
+
+/**
+ * Returns a function that returns true if the element has a specific name.
+ * @param {string} name The name to check.
+ * @returns {function(*)} A function that returns true if the element has a specific name.
+ */
+export function elementHasName( name ) {
+	return ( elementThing ) => {
+		return elementThing.name === name;
```

Comment:
elementThing is not a nice name for a variable, but `element` is already a thing in javaScript.

Suggestions are welcome.